python/hsfs/feature_group.py

```diff
@@ -1270,9 +1272,14 @@ def _create_statistics_monitoring(
             feature_name: Name of the feature to monitor. If not specified, statistics
                 will be computed for all features.
             job_frequency: Frequency at which to compute the statistics for the feature.
-                Options are "HOURLY", "DAILY", "WEEKLY", "MONTHLY", defaults to "DAILY".
+                Options are "NEAR REAL-TIME", "HOURLY", "DAILY", "WEEKLY", defaults to "DAILY".
```

Comment:
What is NEAR REAL-TIME? How is it handled?
In the JobScheduler constructor, it seems only HOURLY, DAILY and WEEKLY are supported